nixos/modules/services/torrent/transmission.nix

```diff
@@ -174,7 +174,9 @@ in
         };
       };
 
-      package = mkPackageOption pkgs "transmission" {};
+      package = mkPackageOption pkgs "transmission_3" {
+        example = "pkgs.transmission_4";
+      };
```

Comment:
The documentation will make more sense with:

```suggestion
      package = mkPackageOption pkgs "transmission" {
        default = [ "transmission_3" ];
        example = "pkgs.transmission_4";
      };
```

Reply:
> The documentation will make more sense with:

Thanks! Using a `default` seems weird to me, and I get the desired result with a single item instead:

![image](https://github.com/NixOS/nixpkgs/assets/10998835/799f9a20-f930-471e-9a9c-1280299c4b29)